### DISCUSSION TITLE
Map Location

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#23] [Q&A] [answered]
I think the map download should end up in the location: Android/Media/Maps!?
I found the downloaded ones here: Android/data/gr.talent.cruiser/files/Download ?
As a .zip file

**Comment by devemux86** [comment]
This Download folder is the default location set by Android for the downloads.
The downloads are handled internally by Android, the apps receive the result.

Did you exit Cruiser while downloads were running?
(you can put the app in background)

After Cruiser download manager finishes a download,
it unzip / moves it to the location selected in "Settings | Application | Storage folder".

Do you mean you have zip files in the Download folder?
Perhaps we should clear such unfinished downloads when app starts.

**Reply by Leeb64** [reply]
I must have quit Cruiser while downloading the map.
Next time I'll leave the app in the background.

**Comment by devemux86** [comment, marked as answer]
> Perhaps we should clear such unfinished downloads when app starts.

It will be available in the next Cruiser update.
Thanks for the report!
